README.md

```diff
@@ -1,2 +1,28 @@
 # actions
 GitHub Actions for earthly
+
+Here's an example of how to use earthly in github actions:
+
+
+```yml
+name: GitHub Actions CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  tests:
+    name: example earthly test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: earthly/actions/setup-earthly@main
+        with:
+          earthly-version: '0.5.10'
```

Comment:
Should we include the `v` here? Then it would just match the tags we use in our repo - easier if we don't assume anything about the version string.

Reply:
the only reason I omitted it, is because we omit it when doing a `brew install earthly@0.5.10`. I can add the `v` prefix in here and remove it for the brew install support. What are your thoughts on that @vladaionescu ?

Reply:
Yeah that makes sense. Not sure if anyone will ever use from Mac anyway.